refactored start() (issue #4)

davidtcalabrese opened the pull request:
Changed start() to use numerical input like other functions.

davidtcalabrese committed:
refactored start() (issue #4)

EmbeddedMan commented on MinnetonkaGame.py:
```diff
@@ -141,23 +141,28 @@ def bird_room():
-  elif "r" in answer:
-    # else if player typed "right" or "r" lead him to big_room()
+  elif answer == "3":
+    # else if player typed "3"lead him to big_room()
```
Need space after "3" in comment

davidtcalabrese replied:
fixed with commit 73f47f6

davidtcalabrese committed:
fixed spacing in comment 

added space after the 3 on line 165 in response to review of PR #11 (issue # 4)

EmbeddedMan reviewed (approved):
Perfect.
